inspirehep/dojson/hepnames/fields/bd1xx.py

```diff
@@ -31,7 +31,6 @@
 
 @hepnames.over('acquisition_source', '^541[10_].')
 @utils.for_each_value
-@utils.filter_values
```

Comment:
:+1: indeed, if some commit is about fixing a previous commit that wasn't yet merged, it's much better to rebase and fixup, i.e. to squash together the commit and the specific correction.